src/Tools/dotnet-user-jwts/src/Helpers/JwtAuthenticationSchemeSettings.cs

```diff
@@ -13,8 +13,10 @@ internal sealed record JwtAuthenticationSchemeSettings(string SchemeName, List<s
     private const string AuthenticationKey = "Authentication";
     private const string SchemesKey = "Schemes";
 
-    private static readonly JsonSerializerOptions _jsonSerializerOptions = new JsonSerializerOptions
+    private static readonly JsonSerializerOptions _jsonSerializerOptions = new()
     {
+        AllowTrailingCommas = true,
```

Comment:
Do I understand correctly that the new options will be ignored during serialization (as opposed to deserialization)?

Reply:
Given the duplication below, I wonder if it would make more sense to pull out and share deserialization options and leave these as-is.

Reply:
My understanding is that these two extra options only apply to deserialisation.

I did consider moving the options to a central place so there was 1 not 4, but wasn't sure _where_ to put that one.

Reply:
I'd be okay with putting them in their own file in the Helpers folder.  My concern is that they could get out sync and then the JSON rules would vary from place to place.